setup.py

```diff
@@ -45,6 +45,7 @@ def run_tests(self):
     include_package_data=True,
     install_requires=[
         'cached-property>=1.3.0, <2',
+        'chronos-python==0.35.0',
```

Comment:
Could this be a wider range? Not blocking but if possible `setup.py` should include a wider acceptable range and `requirements.txt` should contain pinned versions.